src/cfnlint/maintenance.py

```diff
@@ -2,63 +2,19 @@
 Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 SPDX-License-Identifier: MIT-0
 """
-import fnmatch
 import json
 import logging
-import multiprocessing
 import os
-import jsonpointer
-import jsonpatch
+
 import cfnlint
-from cfnlint.helpers import get_url_content, url_has_newer_version
-from cfnlint.helpers import SPEC_REGIONS
+from cfnlint.specs import get_url_content
 import cfnlint.data.ExtendedSpecs
 import cfnlint.data.AdditionalSpecs
 
 
 LOGGER = logging.getLogger(__name__)
 
 
-def update_resource_specs():
```

Comment:
thought it made sense and helped figuring out the [~weekly script](https://github.com/aws-cloudformation/cfn-python-lint/pull/1480#issuecomment-619308734) that they were all in this file or the [`scripts/`](https://github.com/aws-cloudformation/cfn-python-lint/tree/master/scripts) directory

Reply:
Yeah I imagined that was why the code was placed here. But it's not very object-oriented this way. I think it could be better if we documented the process:

```bash
pip3 install -e .
scripts/update_specs_from_pricing.py # requires Boto3 and Credentials
scripts/update_specs_services_from_ssm.py # requires Boto3 and Credentials
cfn-lint --update-specs
cfn-lint --update-iam-policies
cfn-lint --update-documentation
```

BTW the last 2 commands are not explained in the readme's command line options, is that intended?

Reply:
Not really, there were at one point documented.  They ended up being considered to be more admin/maintainer style commands so they ended up being undocumented.  So many commands... `--update-all`?

Reply:
> So many commands... --update-all?

But customers won't care about `--update-documentation`...

Reply:
Maybe we can have a Github action that runs on every PR? It would run `--update-documentation` and if it detected changes it would submit another PR